packages/extension-dapp/src/index.ts

```diff
@@ -124,35 +142,43 @@ export async function web3Accounts ({ ss58Format }: Web3AccountsOptions = {}): P
 
   const addresses = accounts.map(({ address }): string => address);
 
-  console.log(`web3Accounts: Found ${accounts.length} address${accounts.length !== 1 ? 'es' : ''}: ${addresses.join(', ')}`);
+  console.log(
+    `web3Accounts: Found ${accounts.length} address${accounts.length !== 1 ? 'es' : ''}: ${addresses.join(', ')}`
+  );
 
   return accounts;
 }
 
-export async function web3AccountsSubscribe (cb: (accounts: InjectedAccountWithMeta[]) => void | Promise<void>, { ss58Format }: Web3AccountsOptions = {}): Promise<Unsubcall> {
+export async function web3AccountsSubscribe (
+  cb: (accounts: InjectedAccountWithMeta[]) => void | Promise<void>,
+  { ss58Format }: Web3AccountsOptions = {}
+): Promise<Unsubcall> {
```

Comment:
Change to the rest of this file seems unneeded (after spending some time to go through). Would suggest just keeping to the logic to make reviewing slightly easier. (Not a critisism, but beauty is in the eye of the beholder and since tastes differ, it does make it much harder to get to the actual meat than it should be here - if reformatting is needed, would suggest making a different PR which just focusses on that)

Reply:
I agree but I don't know how to revert this formatting that comes from the linter

Reply:
Oh I guess I could run this PR against a formatted version 👍